src/Language/Haskell/Exts/InternalLexer.hs

```diff
@@ -99,6 +99,8 @@ data Token
         | RightArrowTail        -- >-
         | LeftDblArrowTail      -- -<<
         | RightDblArrowTail     -- >>-
+        | OpenArrowBracket      -- (|
+        | CloseArrowBracket     -- |)
```

Comment:
Would it make sense to name these `OpenBananaBracket`, `CloseBananaBracket`? I think those names will be a little more obvious to people.

Reply:
I think that’s reasonable, though the only reservation I have is that the HaRP syntax also has banana brackets, which appear as a different token. Do you think it’s okay to use “banana bracket” to refer to the arrow versions exclusively?

Reply:
Gave it some thought; your original choice of `OpenArrowBracket` an `CloseArrowBracket` are good.